onset_functions.py

```diff
@@ -1,6 +1,7 @@
 
 import os
 import datetime
+from turtle import position
```

Comment:
What is this for? It's not used (similar to `from turtle import speed` that I uncommented before)

Reply:
I have no idea where that came from. Actually, why is turtle imported at all when it's not used anywhere?

Reply:
What is turtle actually?

Reply:
Can this be some code-completion in VS Code that occurs when you e.g. type 'position', so that it automatically adds some import it thinks you want?

Reply:
Turtle is python's built-in graphical drawing and object editor. I see no use for it here. 

VS Code automatic code completion sounds like a plausible reason for the addition. I'll delete it now, since it was an unintended import anyway.